personal_profile/tests.py

```diff
@@ -31,13 +31,30 @@ def profile_1(db, user_1):
     return profile_1
 
 
+@pytest.mark.django_db
 class TestProfileModel:
-    # Testing different Personal Profile attributes
+    # Testing different Personal Profile saving and deletation
 
-    def test_newprofile(self, profile_1, user_1):
+    def test_save_newprofile(self, profile_1, user_1):
         assert profile_1.company == 'Test Company'
```

Comment:
Please take it as general comment
The test should filter the data and compare it to fixtures returned values

Reply:
@Yarboa what do you mean by "filter the data"? Thanks!

Reply:
Currently there is a call to fixture, enter data to db,
then the same objects are asserted vs Text.

Instead of searching in db and convert the fixture content, i hope it is clear